src/components/BotBuilder/BotBuilder.js

```diff
@@ -135,6 +137,76 @@ class BotBuilder extends React.Component {
     });
   };
 
+  getDrafts = () => {
+    let url = BASE_URL + '/cms/readDraft.json';
+    $.ajax({
+      url: url,
+      dataType: 'jsonp',
+      crossDomain: true,
+      success: function(data) {
+        this.showDrafts(data.drafts);
+      }.bind(this),
+      error: function(error) {
+        this.setState({
+          openSnackbar: true,
+          msgSnackbar: "Couldn't get your drafts. Please reload the page.",
+        });
+      }.bind(this),
+    });
+  };
+
+  showDrafts = drafts => {
+    if (drafts) {
+      let draftsOfBots = [];
+      for (let draft in drafts) {
+        draftsOfBots.push(
+          <Card key={drafts[draft].name} className="bot-template-card">
```

Comment:
Please use the `id` of the draft as the `key` here. If the user is saving more than one bot with blank name, only one of them is appearing here. So we need to use `id` as the key.